src/app/containers/ConsentBanner/Canonical/logic/cookieOvenBaseUrl.js

```diff
@@ -0,0 +1,20 @@
+const getCookieOvenBaseUrl = origin => {
+  let base = 'https://cookie-oven.api.bbc';
+  let tld = '.com';
+
+  if (origin.includes('localhost')) {
+    return origin;
```

Comment:
This would read more easily if you just had two functions  `cookieOvenDomainBase`
`domainExtension` 
Then concatenating them on the final line.

Reply:
Done :)